packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -80,6 +80,8 @@ export default class ControllerInterface {
 
     const swReg = await this.getSWRegistration_();
     const publicVapidKey = await this.getPublicVapidKey_();
+    // If there didn't exist one, this call will subscribe a new
+    // push subscription.
```

Comment:
Re-word to:

If a PushSubscription exists it's returned, otherwise a new subscription is generated and returned.

Reply:
done